docs/android/customisation/loyalty.md

```diff
@@ -100,4 +103,31 @@ This can be changed to large as follows:
     //..
     <item name="ctLoyaltyChipSize">regular</item> // Either "regular" or "large" Default is "regular"
 </style>
+```
+
+## Loyalty Banner Landing Screen
+The template of the loyalty banner on the landing screen can be modified. There are two templates available "Default" and "Logo & Text". The later is a full width text with a central logo and an optional gradient background.
+
+<img src="/uploads/loyalty_banner_templates.png">
+
+The Logo & Text template can be styled as follows:
+
+Create a style for the banner:
+```xml
+    <style name="ctLoyaltyLandingBannerStyle">
+        <item name="ctBannerTemplate">logoAndTextBanner</item> // Either "defaultBanner" or "logoAndText". Default is "defaultBanner"
+        <item name="ctBannerBackgroundColor">?colorPrimary</item> // Defaults to the theme's colorPrimary
+        <item name="ctBannerSecondaryBackgroundColor">?colorSecondary</item> // Defaults to the theme's colorSecondary
+        <item name="ctBannerTextColor">@android:color/white</item> // Defaults to the Android's color white 
```

Comment:
I would reword the comment for **ctBannerTextColor** and **ctBannerInfoButtonColor** to "Defaults to Android's white color"